src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -45,12 +45,17 @@
       <HintPath>..\..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="PresentationCore" />
```

Comment:
so none of these references should be added to CoreNodeModels, including windows base.